### ISSUE TITLE
Allow users to open table links inside of a cell by pressing Ctrl + Click

### ISSUE THREAD

**Issue by adam2am** [Zettlr/Zettlr#4694]
### Checklist

- [X] Zettlr does not possess this feature (if it does, but the feature doesn't work out for you, propose an enhancement instead.)
- [X] Zettlr's current features cannot be used to approximate this behaviour (otherwise, propose an enhancement instead.)
- [X] I have searched the documentation to make sure I did not overlook it.
- [X] I have searched both open and closed issues to see if someone has already proposed this.

### Feature Proposal

Currently, when I try to click on a link in a table cell while holding down the Ctrl key, Zettlr still edits the cell instead of opening the link. This can be inconvenient, especially if I have a lot of links in my table.

I would like to request a new feature that would make table links clickable when pressing Ctrl. This feature would be especially useful for users who work with a lot of tables and links. For example, researchers, writers, and students often need to work with tables of data that contain links to sources. This feature would allow them to quickly open the sources in their browser without having to leave their table.

### Example Workflow

A students are writing a research paper in Zettler. They have created a table to summarize the findings of the research papers they have cited. Each table cell contains a link to the corresponding research paper.

The student wants to quickly open all of the research papers in their browser so that they can review them before finishing their paper. To do this, they can hold down the Ctrl key and click on each link in the table. This will open each research paper in a new browser tab.

The students can then review the research papers and add any additional information they need to their paper.

### Caveats

It could conflict with the existing keyboard shortcut for editing table cells. Currently, pressing Ctrl and clicking on a table cell will edit the cell. If the new feature is implemented, pressing Ctrl and clicking on a table cell could either open the link or edit the cell, depending on the user's settings

### Do you wish to attempt implementing this yourself?

No

### Platforms

- [X] Windows
- [ ] macOS
- [ ] Linux

**Comment by jefferys** [comment]
I am a new user considering moving to Zettlr from Obsidian due to this being open source. This behavior is unexpected. It breaks my workflow for many of my existing documents. I can command-click on links open for editing as well as rendered links if in body text. I can imagine the within table editor code, it might be complicated to support command clicking on links being edited, but the rendered table cell should be able to open the link when command clicking; it is not just a plain click, the table editor should not be invoked?

Note, I am explicitly interested in markdown links working, not wiki links.

**Comment by adam2am** [comment]
> I am a new user considering moving to Zettlr from Obsidian due to this being open source. This behavior is unexpected. It breaks my workflow for many of my existing documents. I can command-click on links open for editing as well as rendered links if in body text. I can imagine the within table editor code, it might be complicated to support command clicking on links being edited, but the rendered table cell should be able to open the link when command clicking; it is not just a plain click, the table editor should not be invoked?
> 
> Note, I am explicitly interested in markdown links working, not wiki links.

Yeah, this issue has been open since 2023, so I wouldn’t expect a maintainer fix anytime soon. You may want to vibecode it quick/implement this yourself and open a PR if it’s important for your workflow

**Comment by nathanlesage** [comment]
> Yeah, this issue has been open since 2023, so I wouldn’t expect a maintainer fix anytime soon.

I don't know where you take this from, but typically, the older requests are, the more likely they become to be implemented. This one has been on my radar, but we've had some other issues to fix, and because no one commented here, it dropped off my radar again. Should be a quite straight-forward fix.
